webextensions/api/storage.json

```diff
@@ -378,52 +582,442 @@
+              "support": {
+                "chrome": {
+                  "version_added": "≤58",
+                  "notes": "Always resolves with a value of 0."
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "onChanged": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/onChanged",
+              "support": {
+                "chrome": {
+                  "version_added": "73"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "101"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "remove": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/remove",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "57"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "empty_key": {
+              "__compat": {
+                "description": "Supports empty key",
+                "support": {
+                  "chrome": {
+                    "version_added": "≤105"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "57"
+                  },
+                  "firefox_android": {
+                    "version_added": false
+                  },
+                  "opera": {
+                    "version_added": false
+                  },
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
```

Comment:
Drop `empty_key` from `storage.managed.remove`; it was a Safari-only issue, but storage.managed is not supported in Safari so not relevant.

```suggestion
```